code/test/VsTemplates.Test/VsTemplatesContentValidationTests.cs

```diff
@@ -0,0 +1,160 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using VsTemplates.Test.Validator;
+using Xunit;
+
+namespace VsTemplates.Test
+{
+    [Trait("Type", "VsTemplateValidation")]
+    [Trait("ExecutionSet", "Minimum")]
+    [Trait("ExecutionSet", "VsTemplateValidation")]
+    [Trait("ExecutionSet", "_CIBuild")]
+    [Trait("ExecutionSet", "_Full")]
+    public class VsTemplatesContentValidationTests
+    {
+        private const string ItemTemplatesRoot = "../../../../../src/ItemTemplates";
+        private const string ProjectTemplatesRoot = "../../../../../src/ProjectTemplates";
+        private const string VsTemplateFileExtension = ".vstemplate";
+        private static readonly List<string> WizardProjectTemplatesRoot = new List<string>
+        {
+            "../../../../../src/ProjectTemplates/UWP",
+            "../../../../../src/ProjectTemplates/WPF",
+            "../../../../../src/ProjectTemplates/WinUI/Cpp/Cpp.WinUI.Desktop.Solution",
```

Comment:
Do you really need to specify so many subdirectories? Couldn't you just use "../../../../../src/ProjectTemplates/WinUI/"?

Reply:
These directories include projects that open the WTS wizard. In "../../../../../src/ProjectTemplates/WinUI/" directory we can find other project templates like ClassLibrary or RuntimeComponent that doesn't run the wizard